tests/samples_tests/smoke_tests/requirements.txt

```diff
@@ -1,5 +1,7 @@
 -c ../../constraints.txt
-requests
-pyyaml
 numpy
+opencv-python-headless
+requests
 pytest
+pytest-xdist  # Enable pytest --numprocesses
```

Comment:
```suggestion
pytest-xdist[psutil]  # Enable pytest --numprocesses
```

Reply:
My intention was to avoid `psutil`, because I can't tell anyway which is faster: `auto` or `logical`. Sometimes one is faster, sometimes another. But now I see that `xdists`'s description is controversial here:
`--help` prints
> With 'auto', attempt to detect physical CPU count. With 'logical', detect logical CPU count. If physical CPU count cannot be found, falls to logical count.

and https://pytest-xdist.readthedocs.io/en/stable/distribution.html states
> Use -n logical to use the number of logical CPU cores rather than physical ones. This currently requires the psutil package to be installed; if it is not, pytest-xdist will fall back to -n auto behavior.

So auto falls to logical and logical falls to auto. I'll submit an issue to them

Reply:
Currently it is not important because GHA runners are docker containers (physical cores == logical cores) and it works as -n auto